acceptance/launcher_test.go

```diff
@@ -31,12 +33,9 @@ func testLauncher(t *testing.T, when spec.G, it spec.S) {
 		when("CNB_PROCESS_TYPE is NOT set", func() {
 			it("web is the default process-type", func() {
 				cmd := exec.Command("docker", "run", "--rm", launchImage)
-				output, err := cmd.CombinedOutput()
-				if err != nil {
-					t.Fatalf("failed to run %v\n OUTPUT: %s\n ERROR: %s\n", cmd.Args, output, err)
-				}
+				output := h.Run(t, cmd)
```

Comment:
It is a great change! Love reducing code duplication :)
Two small things about it:
1. `cmd.CombineOutput` checks that `Stdout` and `Stderr` are `nil` before calling to `Run`. Do you think we should add it once at the beginning of `testLauncher`?
2. Do you think it's worth creating a function that combines both `Run` and checks whether the `output` contains some expected strings?

Reply:
For 1, I don't think it is necessary to check, as `h.Run()` calls `cmd.Output()` which also has this check for stdout, and we initialize stderr to empty. I think this is more about the internals of how the `cmd` is being run, vs. what we are testing.

For 2, I believe the windows PR that's currently up will introduce an `assertOutput`. We might take advantage of that - I didn't do it here as some of the assertions are using regex matchers which don't really pair nicely with that kind of matcher. But maybe we can have something like `assertOutputMatches`...

Reply:
I am going to hold off on making these changes for now - for launcher_test.go, let's wait for the Windows PR. For this PR, I am not sure if the `assertOutput` matcher makes sense, as it wraps the call to `cmd.CombinedOutput()` and we really only want to call that once. We have plenty of test cases that make multiple assertions on the output - such as here: https://github.com/buildpacks/lifecycle/pull/319/files#diff-72171b9647f1d25c895f3dcfa8e02b44R208-R214 (note `AssertNotMatch` mixed with `AssertMatch`) and I am not sure how to do that cleanly; it seems the effort required might not be justified by the gains in reduced duplication and/or clarity. @yaelharel let me know if you feel strongly about it or have another suggestion.

Reply:
I'm ok with keeping the code as is for now. We can revisit it later.